src/redux/store.js

```diff
@@ -0,0 +1,11 @@
+import { configureStore } from '@reduxjs/toolkit';
+import RocketSlice from './rockets/RocketSlice';
+import MissionSlice from './missions/MissionSlice';
+
+const store = configureStore({
+  reducer: {
+    rocket: RocketSlice,
+    mission: MissionSlice,
+  },
+});
+export default store;
```

Comment:
- [ ] it would be better to use `redux-logger` as required in order
<img width="505" alt="Screenshot 2023-04-13 at 17 31 36" src="https://user-images.githubusercontent.com/61790587/231838749-32c3d719-f0a6-4aea-8a59-eac5d107499c.png">
 to complete the requirements

Reply:
✅ As requested by the previous code reviewer please consider adding the REDUX LOGGER MIDDLEWARE to your store to meet the project requirements.

## MORE INFO:-

You can have a look at the following code to understand it better:-

```
import { applyMiddleware, createStore } from 'redux';
 
// Logger with default options
import logger from 'redux-logger'
const store = createStore(
  reducer,
  applyMiddleware(logger)
)
 
// Note passing middleware as the third argument requires redux@>=3.1.0
```

Reply:
- [ ] Please kindly implement this as well, I noticed that the second reviewer had explained how to address it. We normally use Redux middleware for logging, crash reporting, talking to an asynchronous API, routing, and more. ℹ️